applications/minotari_console_wallet/src/automation/commands.rs

```diff
@@ -803,17 +803,29 @@ pub async fn command_runner(
             },
             ExportUtxos(args) => match output_service.get_unspent_outputs().await {
```

Comment:
I would prefer it if we add new methods, rather than changing the existing to expose secrets. 

Or make the secret key expose part optional.